app/src/main/java/com/karrar/movieapp/ui/composable/MediaItem.kt

```diff
@@ -0,0 +1,32 @@
+package com.karrar.movieapp.ui.composable
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
+
+@Composable
+fun MediaItem(
+    modifier: Modifier = Modifier,
+    mediaImageUrl: String,
+    imageHeight: Int = 176,
+    onClick: () -> Unit
+) {
+    Image(
+        painter = rememberAsyncImagePainter(model = mediaImageUrl),
+        contentDescription = null,
+        contentScale = ContentScale.Crop,
+        modifier = modifier
+            .wrapContentSize()
+            .height(imageHeight.dp)
+            .width((imageHeight + 50).dp)
```

Comment:
why have height and add 50?

Reply:
the relation that I found between diffrent these two  (128*176), (99*143)

<img width="178" alt="Screenshot 2023-03-15 at 4 41 53 PM" src="https://user-images.githubusercontent.com/34461597/225468396-fb0337c3-586c-42ac-9dd6-e844378a5592.png">

<img width="280" alt="Screenshot 2023-03-15 at 4 41 41 PM" src="https://user-images.githubusercontent.com/34461597/225468456-e3c9baad-8bdf-4a8a-b83b-4fd7c2ff71c4.png">